### PR TITLE
(WIP, DO NOT MERGE) Replay drag n drop

### PR DESCRIPTION
**Pull request by SuperV1234**
Major features:
- Automatically save local best replays to `./Replays`
- Drag and drop a replay file on the `SSVOpenHexagon.exe` executable to immediately view it

Currently not working properly, for some reason:
- When playing Arcadia, any saved replay will work correctly after exiting Open Hexagon and dropping the newly saved replay on it
- When playing other levels, such as Pointless, the replay will be incorrect on the first execution (right after dropping it on the executable), but pressing Y to execute it again will play it correctly
- There must be some issue with either the RNG seed or with the state of the level that is only fixed after the replay is executed one more time, but I couldn't figure it out yet... help appreciated

### CODE REVIEW

**Comment by Morxemplum** [comment]
Should label this as a draft if you do not want it to be merged

**Comment by SuperV1234** [comment]
@Morxemplum: done. Let me know if you're willing to spend some time debugging the issue, I can walk you through it

**Comment by Morxemplum** [comment]
Lol that's not what I meant by making it a draft. You can convert a pull request to a draft to show it's WIP and you shouldn't merge it. But after a Sound Update PR I'd like to try and take a look at it, but I can't guarantee a solution

**Comment by SuperV1234** [comment]
Oh I see, lol.

File: src/SSVOpenHexagon/Core/main.cpp [hunk 1]
```diff
@@ -2,110 +2,116 @@
...
-    {
-        hg::Config::loadConfig(overrideIds);
+    return std::make_tuple(result, cliLevelName, cliLevelPack);
+}
+
+static std::string makeWindowTitle()
+{
+    return "Open Hexagon " + std::string{hg::Config::getVersionString()} +
+           " - by vittorio romeo - http://vittorioromeo.info";
+}
```
Comment (Morxemplum) [inline src/SSVOpenHexagon/Core/main.cpp:71]: *sad noises*

**Pull request merged**